draft-meynell-panrg-scion-research-questions.md

```diff
@@ -470,12 +474,27 @@ Is reverse path refresh a relevant problem?
 * What time synchronization precision should we expect at the border router level of every AS? How far can we go realistically?
 -->
 
-# Interfaces for Path Awareness
 
-* IPv6 in the Data Plane
-* SCION-IP translation
+# Interfacing SCION with Existing Technologies
+The questions posed here are:
+
+* What existing protocols/solutions should be compatible with SCION simultanously?
+  Referring to the probable fact that ISPs want to offer traditional IP services aside from SCION.
+* What existing technologies can replace part of the SCION specification?
+  Referring to the possibility of slightly changing an existing technology (e.g. IPv6) to be used
+  as part of SCION, replacing part of the ad-hoc specification that we have for SCION.
+* What would be required effort to make them work?
+  Referring to the ranking according to different types of parties involved: ISPs, vendors,
+  application developers, etc.
```

Comment:
Is this about deployment incentives? Or Implementation effort?

Reply:
Incentives vs. effort, for the different parties.